src/Microsoft.Owin.Security.Facebook/Constants.cs

```diff
@@ -7,8 +7,8 @@ internal static class Constants
     {
         public const string DefaultAuthenticationType = "Facebook";
 
-        internal const string AuthorizationEndpoint = "https://www.facebook.com/v2.8/dialog/oauth";
-        internal const string TokenEndpoint = "https://graph.facebook.com/v2.8/oauth/access_token";
-        internal const string UserInformationEndpoint = "https://graph.facebook.com/v2.8/me";
```

Comment:
Why did you remove the version field? All the samples still include it (v3.2)
https://developers.facebook.com/docs/facebook-login/manually-build-a-login-flow#login

Reply:
"If your call doesn't specify a version it will default to the oldest available version."

Reply:
I figured it would default to either newest or oldest. 

Depends on if the approach is to lock it to a particular version and then the need to remember to update it in future when Facebook deprecates that version is still there or change it so it just reflects latest version always so doesn't require an update?

There are pros/cons to each approach, I am not sure what is the preference for this package?

Reply:
We make a point of using a specific version so we can periodically verify the latest functionality and it will stay the same for two years. If that version goes obsolete before we update it then it automatically rolls forward to the next version similar to if we had not specified a version.

Reply:
At this point it has been stable for several versions so I would not anticipate updating it unless we needed to react to a specific service change.